lib/pdf/inspector/text.rb

```diff
@@ -62,6 +64,11 @@ def set_word_spacing(*params)
         @word_spacing << params[0]
       end
       
+      def set_horizontal_text_scaling(*params)
```

Comment:
That crossed my mind, actually. It may be the case for other operators. But as far as I can tell from PDF 1.7 spec `Tm` takes exactly 6 arguments. As well as other text-related operators take fixed number of arguments.

In this case we only use the first argument anyway (which, I suppose is an array of number). So why do we use more generic code and fiddle with argument extraction is we can have just one argument (or six)?